notify/wechat/wechat.go

```diff
@@ -132,7 +136,8 @@ func (n *Notifier) Notify(ctx context.Context, as ...*types.Alert) (bool, error)
 
 		// Cache accessToken
 		n.accessToken = wechatToken.AccessToken
-		n.accessTokenAt = time.Now()
+		n.accessTokenAt = startTime
+		n.expireIn = time.Duration(wechatToken.ExpiresIn) * time.Second
```

Comment:
instead of storing the creation time of the token and its period, you could record its expiration time (minus a safety interval as described above).